pkg/reconciler/v1alpha1/configuration/configuration.go

```diff
@@ -276,6 +276,7 @@ func (c *Reconciler) createRevision(ctx context.Context, config *v1alpha1.Config
 			LabelSelector: fmt.Sprintf("%s=%s", serving.BuildHashLabelKey, buildHash),
 		})
 		if err != nil {
+			logger.Errorf("Failed to list GroupVersionResource %+v: %v", gvr, err)
```

Comment:
To cut down on double logging, should we just wrap the error here and let the caller print the error? The call to `createRevision` already prints an error if one is returned, so we could add some context here by:

```golang
import errutil "github.com/pkg/errors"

return nil, errutil.Wrap(err, "failed to list GroupVersionResource")
```

Does that make sense?

Reply:
Yeah. Makes sense. Thanks